charts/datadog-crds/update-crds.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+set -euo pipefail
+
+ROOT=$(git rev-parse --show-toplevel)
+
+TAG=master
+if [[ $# -eq 1 ]] ; then
+    TAG=$1
+fi
+
+download_crd() {
+    inFile=datadoghq.com_$2_crd.yaml
+    version=$4
+    outFile=datadoghq.com_$2_$version.yaml
+    path=$ROOT/charts/datadog-crds/templates/$outFile
+    echo "Download CRD \"$inFile\" version \"$version\" from tag \"$1\""
+    curl --silent --show-error --fail --location --output $path https://raw.githubusercontent.com/DataDog/datadog-operator/$1/deploy/crds/$version/$inFile
```

Comment:
Source of truth is now `config/crd/bases/`

Reply:
PR updated